classes_part1.py

```diff
@@ -0,0 +1,61 @@
+import datetime
+import time
+import pet
+
+# Create empty array to store instances of pets in
+pet_list = []
+
+# Get info about Pet 1 and store it
+print("Let's start by gathering basic data about your first pet.\n")
+pet_1 = pet.pet('','','')
+pet_1.getName()
+pet_1.getCategory()
+pet_1.getSubtype()
+#   Add Pet 1 to array
+pet_list.append(pet_1)
+
+# Get info about Pet 2 and store it
+print("\nNow let's gather more detailed data about your second pet.\n")
+pet_2 = pet.pet('','','')
+pet_2.getName()
+pet_2.getCategory()
+pet_2.getSubtype()
+pet_2.getBirthdate()
+pet_2.getColors()
+pet_2.getHasTail()
+pet_2.getHasWings()
+pet_2.getNumOfLegs()
+pet_2.getLastFed()
+#   Add Pet 2 to array
+pet_list.append(pet_2)
+
+# Hard code info for Pet 3 and Pet 4
+print("\nI know you have also have 2 fish, Flotsam and Jetsam.  Let me enter the info I know about them now...")
+time.sleep(2)
+#   Add Pet 3 to array
+pet_3 = pet.pet('Flotsam', 'fish', 'goldfish', '', '', '', '', '', datetime.date.today() - datetime.timedelta(days=1), '')
+pet_list.append(pet_3)
+#   Add Pet 4 to array
+pet_4 = pet.pet('Jetsam', 'fish', 'betafish', '', '', '', '', '', datetime.date.today() - datetime.timedelta(days=10), '')
```

Comment:
Same as pet_3, don't call the constructor and pass in '', '', '', '' etc. Instead, call it with the first 3 arguments, then call setLastFed.

Or, use argument names (Which are pretty cool)

pet_4 = pet.pet(name = 'Jetsam', category = 'fish', subtype = 'betafish', last_fed = datetime.date.today() - datetime.timedelta(days=10))